en-US/firefox-ios.xliff

```diff
@@ -2368,10 +2365,45 @@
         <target>Share Link</target>
         <note>Context menu item for sharing a link URL</note>
       </trans-unit>
-      <trans-unit id="ShareTo.Cancel.Button">
```

Comment:
@garvankeeley would it not be better to rename `SendTo.Cancel.Button` to `ShareTo.Cancel.Button` - since the SendTo extension does not exist anymore?

Reply:
Yes, I didn't want to mess up existing localizations so I avoided changing the SendTo strings in any way.